src/comment-pr.ts

```diff
@@ -6,6 +6,7 @@ import {RequestError} from '@octokit/request-error'
 import {ConfigurationOptions} from './schemas'
 
 export const MAX_COMMENT_LENGTH = 65536
+export const MAX_GITHUB_STEP_SUMMARY_LENGTH = 1024 //1kb
```

Comment:
the summary max is 1048576 bytes, not 1024.  it is 1MiB.  although i don't see where this constant is being used?